jgiven-core/src/main/java/com/tngtech/jgiven/impl/util/PrintWriterUtil.java

```diff
@@ -1,15 +1,24 @@
 package com.tngtech.jgiven.impl.util;
 
-import java.io.*;
-
 import com.google.common.base.Charsets;
 import com.google.common.base.Throwables;
 import com.tngtech.jgiven.config.ConfigValue;
 
+import java.io.File;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+
 public class PrintWriterUtil {
     public static PrintWriter getPrintWriter( File file ) {
         try {
-            return new PrintWriter( file, Charsets.UTF_8.name() );
+            return new PrintWriter( file, Charsets.UTF_8.name() ) {
+                @Override
+                public void println() {
+                    write( "\n" );
```

Comment:
why is that better than the code of the standard `PrintWriter` class? Also, if there is a reason to circumvent the double checked locking in there, wouldn't you at least want to use `this.lineSeparator` instead of a hardcoded `\n`

Reply:
Ah, this is something I quickly did to get the build working on my windows machine. I'll revert and look for something a little less "hacky"